src/corehost/cli/deps_format.cpp

```diff
@@ -68,7 +63,7 @@ void deps_json_t::reconcile_libraries_with_targets(
                 size_t pos = library.first.find(_X("/"));
                 entry.library_name = library.first.substr(0, pos);
                 entry.library_version = library.first.substr(pos + 1);
-                entry.library_type = _X("package");
+                entry.library_type = pal::to_lower(library.second.at(_X("type")).as_string());
```

Comment:
For example, "project" assemblies will not be present in "packages" cache or in the FX base, so these probes are useless. We can directly go to app base. The host doesn't do this optimization today. I'm trying to say the host shouldn't otherwise care about the type in the libraries section.